cmd/scollector/collectors/redis_remote.go

```diff
@@ -0,0 +1,23 @@
+package collectors
+
+import (
+	"fmt"
+
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/opentsdb"
+)
+
+func init() {
+	registerInit(startRedisRemoteCollector)
+}
+
+func startRedisRemoteCollector(c *conf.Conf) {
```

Comment:
not sure why this code could not be combined with the other redis init function and this file would not be needed.